guides/release/code-editors/index.md

```diff
@@ -37,29 +45,53 @@ Both editors share a range of cross-compatible extensions listed below.
 You'll want to remove any linter / completion manager you currently have installed
 (or disable them for `.js`, `.ts` or `.hbs` files), and follow the install guides for the following packages:
 
-[ember.vim](https://github.com/dsawardekar/ember.vim) -
-Shortcuts to navigate related files with Ember.js projects.
+### Syntax Highlighting
+
+Only one of these solutions is needed, with tree-sitter being the highest fidelity.
+
+[nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) -
+high-fidelity static highlighting with support for `.gjs` and `.gts` and embedded `hbs`.
+Use `ensure_installed = { 'glimmer' }`.
+Example `nvim-treesitter` config [can be found here](https://github.com/NullVoxPopuli/dotfiles/blob/main/home/.config/nvim/lua/plugins/syntax.lua#L15)
+
+or
 
 [vim-ember-hbs](https://github.com/joukevandermaas/vim-ember-hbs) -
 Add Ember template syntax highlighting and indentation to Vim.
+To get embedded highlighting will involve these additional plugins:
+- [vim-javascript](https://github.com/pangloss/vim-javascript)
+- [vim-js-pretty-template](https://github.com/Quramy/vim-js-pretty-template)
 
-[Ember Tools](https://github.com/AndrewRadev/ember_tools.vim) -
-Various tools for working with Ember.js projects.
+
+### Language Server
+
+Only one of these solutions should be used at a time
+
+Native LSP with assistance from [Mason.nvim](https://github.com/williamboman/mason.nvim)
+Easily install and manage LSP servers, DAP servers, linters, and formatters.
+Example mason+LSP config [can be found here](https://github.com/NullVoxPopuli/dotfiles/tree/main/home/.config/nvim/lua)
+
+or
 
 [Conquer for Completion (COC) for Neovim](https://github.com/neoclide/coc.nvim) -
 An Intellisense engine which takes control over all linting, hinting, and language-server integration.
-
-[coc-ember](https://github.com/NullVoxPopuli/coc-ember) -
+With the ember plugin [coc-ember](https://github.com/NullVoxPopuli/coc-ember) -
 Ember.js language server extension including useful configuration instructions.
 
-[nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) -
-high-fidelity static highlighting. With `ensure_installed = { 'glimmer' }`, this
-also includes support for embedded templates in JavaScript and typescript.
+[Conquer for Completion (COC) for Neovim](https://github.com/neoclide/coc.nvim) -
+An Intellisense engine which takes control over all linting, hinting, and language-server integration.
+
```

Comment:
This is a repeat of line 76. Can you delete this or the previous mention above?

Reply:
oops. thanks!